News importer + news template

akasjain2 opened the pull request:
- Adds importer for news - Imports News articles, PDF brochures points to /assets/{filename}.pdf. Uploading PDF to sharepoint is pending.
- Adds news template 


Fix [#20](https://github.com/aemsites/octoral/issues/20)

Test URLs:
- Before: https://main--octoral--aemsites.hlx.live/en/news/170-discover-the-ultimate-finish-with-the-universal-surfacer
- After: https://news-template--octoral--aemsites.hlx.live/en/news/170-discover-the-ultimate-finish-with-the-universal-surfacer

dave-fink commented:
@akasjain2 Nice work!!! Here are the things we talked about on our call:
- Adjust import script to identify all caps headings and convert them to h3 - then we can remove that check in the presentation logic
- Adjust tables to use the OOTB Table block - to make authoring easier
- Fix import so that it honors the existing links (hypens and underscores)
- Adjust News heading to be p with a class - we don't want an h2 above an h1 in our code - for accessability purposes.

akasjain2 commented:
> @akasjain2 Nice work!!! Here are the things we talked about on our call:
> 
> * Adjust import script to identify all caps headings and convert them to h3 - then we can remove that check in the presentation logic
> * Adjust tables to use the OOTB Table block - to make authoring easier
> * Fix import so that it honors the existing links (hypens and underscores)
> * Adjust News heading to be p with a class - we don't want an h2 above an h1 in our code - for accessability purposes.

@dave-fink : As discussed, I've addressed these issues in a separate PR having a branch name without '_'. Please review https://github.com/aemsites/octoral/pull/42
